### PR TITLE
CSV header

### LINKED ISSUE #1: Add header to csv files
#### Description
Add proper header to all `.csv` files.

### PR DESCRIPTION
**Pull request by sadrasabouri**
#### Reference Issues/PRs
#1 
#### What does this implement/fix? Explain your changes.
Proper csv headers have been added to csv files.

### CODE REVIEW

**Review by sepandhaghighi — APPROVED** [review]

**Comment by sepandhaghighi** [comment]
@sadrasabouri Well done, LGTM ;-)
@kasraaskari If you are okay with these changes, please merge this `PR` after your approval :-)

**Review by kasraaskari — APPROVED** [review]
@sadrasabouri 
It's done. sorry for delay.

**Comment by sadrasabouri** [comment]
> @sadrasabouri
> It's done. sorry for delay.

Np! :smile:

**Pull request merged**
